sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/checks/BadFunctionNameCheck.kt

```diff
@@ -52,7 +52,8 @@ class BadFunctionNameCheck : AbstractCheck() {
         if (!name.matches(formatRegex)) {
             kotlinFileContext.reportIssue(
                 function.nameIdentifier!!,
-                "Rename function \"$name\" to match the regular expression $format")
+                """Rename function "$name" to match the regular expression: $format"""
```

Comment:
Same as for `BadClassNameCheck`, I suggest removing `:` and perhaps adding `"$format".` here.
```suggestion
                """Rename function "$name" to match the regular expression "$format"."""
```